src/main/java/seedu/address/model/task/TaskPriority.java

```diff
@@ -36,6 +47,23 @@ public int getValue() {
         return taskPriority.getValue();
     }
 
+    /**
+     * Returns true if a given string is a valid description.
+     */
+    public static boolean isValidTaskPriority(String test) {
+        test = test.toLowerCase();
+
+        if (test.equals("low") || test.equals("medium") || test.equals("high")) {
```

Comment:
Should this be equalsIgnoreCase()? If not, we might need to change the user guide.